tasks/strings/strings.js

```diff
@@ -21,16 +18,11 @@ function reverseString(str) {
 */
 function centuryFromYear(year) {
   let arrayNew = year.toString();
```

Comment:
Why so difficult? You do not need to do that.
it can be done in one line:
return Math.ceil(year/100);